src/cmds/restic/cmd_find.go

```diff
@@ -84,7 +85,95 @@ func parseTime(str string) (time.Time, error) {
 	return time.Time{}, errors.Fatalf("unable to parse time: %q", str)
 }
 
-func findInTree(repo *repository.Repository, pat findPattern, id restic.ID, prefix string, snapshotID *string) error {
+type findNode restic.Node
+type omit *struct{}
```

Comment:
"omit" has a certain charm in that it makes clear I do not want to field to show. But sure, I can make them bytes, so the struct becomes as small as possible :)